src/webgl/3d_primitives.js

```diff
@@ -1136,7 +1167,7 @@ p5.RendererGL.prototype.curve = function(
   for (var i = 0; i <= curveDetail; i++) {
     var c1 = Math.pow(i / curveDetail, 3) * 0.5;
     var c2 = Math.pow(i / curveDetail, 2) * 0.5;
-    var c3 = i / curveDetail * 0.5;
+    var c3 = (i / curveDetail) * 0.5;
```

Comment:
```suggestion
    var c3 = i / curveDetail * 0.5;
```
This is why Travis is failing, these brackets are overly verbose. Perhaps we could go for this instead which might be clearer:
```suggestion
    var c3 = 0.5 * i / curveDetail;
```

Reply:
I used the “apply suggestion” feature. I think this is the way the file was. 
I originally made the change because Prettier wasn’t allowing me to commit.